tools/cldr-code/src/main/java/org/unicode/cldr/json/Ldml2JsonConverter.java

```diff
@@ -54,7 +54,8 @@
 @CLDRTool(alias = "ldml2json", description = "Convert CLDR data to JSON")
 public class Ldml2JsonConverter {
     private static final String CLDR_PKG_PREFIX = "cldr-";
-    private static final String FULL_TIER_SUFFIX = "-full";
```

Comment:
Can you explain more about this change and why you're doing it?

Reply:
I think this is an outdated commit